sdk/mediaservices/Azure.ResourceManager.Media/src/Generated/Models/LiveEventInput.cs

```diff
@@ -53,7 +52,7 @@ public IList<IPRange> IPAllowedIPs
         }
 
         /// <summary> ISO 8601 time duration of the key frame interval duration of the input. This value sets the EXT-X-TARGETDURATION property in the HLS output. For example, use PT2S to indicate 2 seconds. Leave the value empty for encoding live events. </summary>
-        public TimeSpan? KeyFrameIntervalDuration { get; set; }
+        public string KeyFrameInterval { get; set; }
```

Comment:
The REST API may return the string '', at which point the SDK will throw `System.FormatException : The string '' is not a valid TimeSpan value`

Reply:
According to the example, the value should be `duration`, but here service return an empty string, so would change the generated code to use `TimeSpan.TryParse` instead of `TimeSpan.Parse`